roles/sriov-nic-init/tasks/load_ddp.yml

```diff
@@ -0,0 +1,34 @@
+# Check if the selected DDP profile is available in Host
+- name: Check that the DDP profile exists
+  stat:
+    path: "{{ ddp_path }}"
+  register: stat_result
+
+- name: Fail if selected DDP profile not found
+  fail:
+    msg: "Selected DDP profile {{ item.ddp_profile }} is not found."
+  when: not stat_result.stat.exists
+
+# Check if we have ethtool
+- name: check ethtool executable is present
+  shell: "which ethtool"
+  register: ethtool_search_result
```

Comment:
maybe instead of checking whether ethtool exists we could just install it using it:
https://github.com/intel/container-experience-kits/tree/master/roles/install-dependencies

it's idempotent, so won't install it if it's already there

this way it would be more "ansible-ish"